### PR TITLE
Change location of generated SSL cert for apiserver

### PR DESCRIPTION
**Pull request by pmorie**
Changes the location of the generated SSL cert to not collide with the core API server.

cc @MHBauer

### CODE REVIEW

File: cmd/apiserver/app/server/server.go [hunk 1]
```diff
@@ -73,6 +77,9 @@ func NewCommandServer(out io.Writer) *cobra.Command {
 	// Store resources in etcd under our special prefix
 	options.EtcdOptions.StorageConfig.Prefix = etcdPathPrefix
 
+	// Set generated SSL cert path correctly
+	options.SecureServingOptions.ServerCert.CertDirectory = certDirectory
```
Comment (MHBauer) [inline cmd/apiserver/app/server/server.go]: I think there's a way to 'override' an existing flag, but I don't know the syntax offhand.

File: docs/api-server-walkthrough.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ kubectl needs a basic kubeconfig setup. A kubeconfig consists of three
 sections that can be set with the following three commands:
 
 1. `kubectl config set-credentials service-catalog-creds --username=admin --password=admin`
-1. `kubectl config set-cluster service-catalog-cluster --server=https://localhost:6443 --certificate-authority=/var/run/kubernetes/apiserver.crt`
+1. `kubectl config set-cluster service-catalog-cluster --server=https://localhost:6443 --certificate-authority=/var/run/kubernetes-service-catalog/apiserver.crt`
```
Comment (MHBauer) [inline docs/api-server-walkthrough.md]: I need to rely on this rather than configuring the kubectl command on each run below.

**Comment by MHBauer** [comment]
@pmorie need https://github.com/kubernetes-incubator/service-catalog/blob/master/contrib/hack/kubectl#L30 for CI. @duglin correct?

File: cmd/apiserver/app/server/server.go (hunk 1 above)
Comment (pmorie) [inline cmd/apiserver/app/server/server.go]: Same, we can do it in a follow-up

**Pull request merged**